Sources/SemanticAnalyzer/SemanticAnalyzer+Declarations.swift

```diff
@@ -550,6 +550,8 @@ extension SemanticAnalyzer {
         if binaryExpression.op.kind == .punctuation(.equal) {
           return true
         }
+      case .typeConversionExpression:
+        fatalError()
```

Comment:
Why `fatalError` here? Given that `isComplexStatement` is only used to make fallbacks simple (???) might as well mark this `true`, since you would only ever use it in an external call.

If you modeled this after `externalCall`, that should also return `true`. Separate issue, I know …

Reply:
I wasn’t sure what this did. I’ll make it return true